packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -324,6 +324,29 @@ export default class VList {
         }
     }
 
+    /**
+     * Get the index of the List Item in the current List
+     * @param input List item to find in the root list
+     */
+    getListItemIndex(input: Node) {
+        if (this.items) {
+            const start = getStart(this.rootList) - 1;
+
+            for (let index = 0; index < this.items.length; index++) {
+                const child = this.items[index];
+                if (
+                    child.getLevel() == 1 &&
+                    !child.isDummy() &&
+                    child.getListType() == ListType.Ordered &&
+                    child.getNode() == input
```

Comment:
do not check getNode() == input here. It will return wrong result from the second child.